app/controllers/public/index.js

```diff
@@ -153,11 +153,7 @@ export default class IndexController extends Controller {
       }
     }
     try {
-      const order = await this.loader.post('/orders/create-order', myinput);
-      this.notify.success(this.l10n.t('Order details saved. Please fill further details within {{time}} minutes.', {
-        time: this.settings.orderExpiryTime
-      }));
-      this.transitionToRoute('orders.new', order.data.attributes.identifier);
+      this.set('orderInput', myinput);
```

Comment:
can you explain  this line.

Reply:
this post request is being handled in save() by placeOrder(). request was being sent two times: once with payload and once without payload(during save()). Now only save will send request which is in pipe action with placeOrder().
https://github.com/fossasia/open-event-frontend/blob/f3ab0cea6561f4b4326ef3e40c12e9496a600b7c/app/templates/public/index.hbs#L55